### DISCUSSION TITLE
School name not recognized on faculty ID card

### DISCUSSION THREAD

**Discussion by galk-research** [community/community#53320] [GitHub Education]
I'm a tenured full professor at Bar Ilan University, in computer science.  I have been twice approved for education discounts by github.  For some reason, could not add github co-pilot addon to the account, so figured I need to refresh my approval.
Twice now, Ive been denied. Once with an image of my faculty ID card, the other with a picture of my holding the card.  Within minutes, I get rejected because my "school name is not recognized".

This is Bar Ilan University, Israel.  The university name and my name are typed out clearly, in English (as well as Hebrew).
There seems to be know email address for support.

Any recommendation as to what I could do?  It seems other teachers are getting similarly stuck (there were some messages from about a month ago), but didn't get any answers here.

**Comment by samus-aran** [comment]
Hi @galk-research 👋 
How did you get on? Did you do anything differently in any later submissions?

**Comment by galk-research** [comment]
It was very frustrating, but it worked on the third or fourth try.  The two
changes (I think):
- I rotated the card image so that it was aligned to the horizontal axis.
- I used a different university email. My primary mail is at the cs
department ***@***.***"), but github kept telling me I should
 use "my university email" at ***@***.***").

It is unfortunate that subdomains of the university are not recognized as
belonging to it.


On Mon, Apr 24, 2023 at 10:43 PM Ashley Nicolson ***@***.***>
wrote:

> Hi @galk-research <https://github.com/galk-research> 👋
> How did you get on? Did you do anything differently in any later
> submissions?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/53320#discussioncomment-5712256>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKV6ICDCRL7KEQTH2W75AS3XC3JV3ANCNFSM6AAAAAAXFWCCEM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
--
-----------------------------------------------------------------------
Gal A. Kaminka, Ph.D.        http://www.cs.biu.ac.il/~galk
Professor    Computer Science Dept.   Bar Ilan University
 Only those who see the invisible can do the impossible

**Reply by samus-aran** [reply]
Thanks for this feedback! Really appreciate you coming back and providing more insight for ourselves and others 👍

**Comment by AzimAhmadzadeh** [comment]
Similar issue here. Most Faculty ID cards are electronic now. They do not have a date. I am being rejected because no date appears on my card. I also tried Syllabus page and employment letter. Still not enough! Any suggestions?

**Comment by doctorjei** [comment]
I am running into the same issue - electronic card reports it is "manipulated". No date, and apparently no support. I've had the academic account for 20 years... I don't know if anyone is every going to fix it. :(
